src/components/Kv/KvContentfulImg.vue

```diff
@@ -6,15 +6,15 @@
 		<!-- Set of image sources -->
 		<template v-if="sourceSizes.length > 0">
 			<template v-for="(image, index) in sourceSizes" :key="index">
-				<!-- browser supports webp -->
+				<!-- browser supports avif -->
 				<source
 					:media="'('+image.media+')'"
-					type="image/webp"
+					type="image/avif"
 					:srcset="`
-					${buildUrl(image.width * 2, image.height * 2)}${crop}&fm=webp&q=65 2x,
-					${buildUrl(image.width, image.height)}${crop}&fm=webp&q=80 1x`"
+					${buildUrl(image.width * 2, image.height * 2)}${crop}&fm=avif&q=65&fl=progressive 2x,
```

Comment:
`fl=progressive` needs to be removed, here and for webp below. webp and avif formats do not support progressive loading and this parameter is not in the contentful image api spec.

Sample image link that does not load: 
https://images.ctfassets.net/j0p9a6ql0rn7/35uESUgMKbnMftC4MTMcw/16bc4f7be98f690cda3eac730891650c/bp-hero-ecuador.jpg?w=1920&h=460&fit=fill&f=center&fm=avif&q=80&fl=progressive